backend/models/user-services.js

```diff
@@ -0,0 +1,149 @@
+const mongoose = require("mongoose");
+const userModel = require("./user");
+const eventModel = require("./events");
+const attendanceModel = require("./attendance");
+const bcrypt = require("bcrypt");
+mongoose.set("debug", true);
+require('dotenv').config();
+
+const conn_str = 'mongodb+srv://ProjectGather:' + process.env.DB_PASSWORD + '@project-gather.iidopil.mongodb.net/?retryWrites=true&w=majority'
+const hashRound = 10;            
+
+mongoose.connect(
+  conn_str,
+  { 
+    useNewUrlParser: true, 
+    useUnifiedTopology: true 
+  },(err) => {
+    if (err) {
+      console.log("error in connection");
+    } else {
+      console.log("mongodb users is connected");
+}})
+
+//WHAT ARE SOME WAYS WE WANT TO FILTER USER BY?
+//--VERIFIED USERS
+async function getUsers() {
+  result = await userModel.find();
+  return result;
+}
+
+async function findUserById(id) {
+  try {
```

Comment:
question: if the function calling findUserById and other user-services functions wraps it in a try-except, is it necessary to have within the function itself, and would that mean that the outer try-except will never trigger for errors?

Reply:
I was doing it to match the other functions, but theoretically yes the outer try except should never be called